test/integration/component/test_ldap.py

```diff
@@ -151,13 +156,18 @@ def test_02_validateldapsecuritypatch(self):
             self.services["configurableData"]["ldap_configuration"])
         self.assertEqual(self.ldapconfRes, 1, "Ldap Configuration failed")
         loginRes = self._checklogin(
-            self.services["configurableData"]["ldap_configuration"]["ldapUsername"], "")
+            self.services["configurableData"]["ldap_configuration"]["ldapUsername"], "","")
         self.assertNotEqual(loginRes, 1, "login API Successful with empty password")
 
     @data("basedn", "ldapPassword")
     def test_03_validateldapbindnobasedn(self, value):
         """
         This test is to verify ldapbind functionality without passing required bind parameters.
+
+        Changes made in September,2015 by Sarath Koundinya : Since ldapbind functionality is moved to Global Settings, Configuring ldap is
```

Comment:
You don't have to mention the date and your name, this info can be retrieved from Git. Please remove it from the comment. The rest of the comment is fine.